systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/ServiceUtils.java

```diff
@@ -79,11 +80,24 @@ public static void waitForNodePortService(String serviceName) throws Interrupted
     public static void waitForServiceDeletion(String serviceName) {
         LOGGER.info("Waiting for Service {} deletion in namespace {}", serviceName, kubeClient().getNamespace());
 
+        kubeClient().deleteService(serviceName);
         TestUtils.waitFor("Service " + serviceName + " to be deleted", Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, DELETION_TIMEOUT,
             () -> kubeClient().getService(serviceName) == null);
         LOGGER.info("Service {} in namespace {} was deleted", serviceName, kubeClient().getNamespace());
     }
 
+    /**
+     * Wait until Service of the given name will be created.
+     * @param serviceName service name
+     */
+    public static void waitForServiceCreation(String serviceName) {
+        LOGGER.info("Waiting for Service {} creation in namespace {}", serviceName, kubeClient().getNamespace());
+
+        TestUtils.waitFor("Service " + serviceName + " to be created", Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, CREATION_TIMEOUT,
+                () -> kubeClient().getService(serviceName) != null);
+        LOGGER.info("Service {} in namespace {} has been created", serviceName, kubeClient().getNamespace());
+    }
+
```

Comment:
You should add the `String namespaceName` to the parameters and work with that

Reply:
Why do I need to add namespace there? It's already there `kubeClient().getNamespace()`.

Reply:
What if we want to wait for service in different namespace? You can add there some encapsulation, but don't forget we have parallel namespaces and the context in the `kubeClient().getNamespace()` can point to completely different namespace.